[package] hdf5/1.14.5: CXXFLAGS are being ignored by the package

### Description

As I wanted to be sure that the version of GCC that clang uses for its header is the right one, I removed from my system the default one.

By default my install of clang tries to use gcc13's header which don't exist as I have removed them.

To make this work I also added in the arguments (tools.build:cxxflags, tools.build:cflags and C/CXXFLAGS) the following: `--gcc-install-dir=/opt/rh/gcc-toolset-12/root/usr/lib/gcc/aarch64-redhat-linux/12 `

Which seems to work flawlessly with the vast majority of Conan packages, except HDF5.

it seems like the `hdf5_cpp-static` target is overriding the cpp flags removing the  `--gcc-install-dir` thus breaking the build process

### Package and Environment Details

* Package Name/Version: **hdf5/1.14.5**
* Operating System+version: **Rocky Linux 8**
* Compiler+version: **Clang 19.1.7**
* Conan version: **2.19.1**
* Python version: **Python 3.12.11**
* CPU architecture: **arm64 neoverse-N2**


### Conan profile

```
[buildenv]
{% set CC = "clang" %}
{% set CXX = CC + "++" %}
{% set CPU = "neoverse-n2" %}

PATH=+(path)/opt/rh/gcc-toolset-12/root/usr/bin/
CC={{CC}}
CXX={{CXX}}
CFLAGS= -mcpu={{CPU}} --gcc-install-dir=/opt/rh/gcc-toolset-12/root/usr/lib/gcc/aarch64-redhat-linux/12 
CXXFLAGS= -mcpu={{CPU}} --gcc-install-dir=/opt/rh/gcc-toolset-12/root/usr/lib/gcc/aarch64-redhat-linux/12 
[conf]
tools.build:cflags=["-mcpu={{CPU}}", "--gcc-install-dir=/opt/rh/gcc-toolset-12/root/usr/lib/gcc/aarch64-redhat-linux/12"]
tools.build:cxxflags=["-mcpu={{CPU}}", "--gcc-install-dir=/opt/rh/gcc-toolset-12/root/usr/lib/gcc/aarch64-redhat-linux/12"]
tools.build:compiler_executables={'c': "{{CC}}", 'cpp': "{{CXX}}" }
tools.cmake.cmakedeps:new=will_break_next
```

### Steps to reproduce

1. Add any compiling option that stands out (for example march/mcpu or in my case --gcc-install-dir) 
2. build the conan pacakge normaly.
3. cd into the package's build folder
4. source the conanbuild.sh
5. make clean
6. make hdf5_cpp-static VERBOSE=1
in the make logs while building H5AbstractDs.cpp (not the only broken file but the first to come up) the flag you added in step 1 disappeared.

### Logs

<details><summary>Failing conan build</summary>

```
hdf5/1.14.5: Building from source
hdf5/1.14.5: Package hdf5/1.14.5:cadfbce37b6206055ba85ecacc65e7f1ab3b640b
hdf5/1.14.5: settings: os=Linux arch=armv9n2 compiler=clang compiler.cppstd=gnu17 compiler.libcxx=libstdc++11 compiler.version=19 build_type=Release
hdf5/1.14.5: options: enable_cxx=True enable_unsupported=False fPIC=True hl=True parallel=False shared=False szip_support=None with_zlib=True with_zlibng=False
hdf5/1.14.5: requires: zlib/1.3.Z
hdf5/1.14.5: Copying sources to build folder
hdf5/1.14.5: Building your package in /home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b
hdf5/1.14.5: Calling generate()
hdf5/1.14.5: Generators folder: /home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/build/Release/generators
hdf5/1.14.5: WARN: Using the new CMakeConfigDeps generator, behind the 'tools.cmake.cmakedeps:new' gate conf. This conf will change next release, breaking, so use it only for testing and dev
hdf5/1.14.5: CMakeDeps necessary find_package() and targets for your CMakeLists.txt
    find_package(ZLIB)
    find_package(cmake) # Optional. This is a tool-require, can't link its targets
    target_link_libraries(... ZLIB::ZLIB)
hdf5/1.14.5: CMakeToolchain generated: conan_toolchain.cmake
hdf5/1.14.5: CMakeToolchain generated: /home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/build/Release/generators/CMakePresets.json
hdf5/1.14.5: CMakeToolchain generated: /home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/src/CMakeUserPresets.json
hdf5/1.14.5: Generating aggregated env files
hdf5/1.14.5: Generated aggregated env files: ['conanbuild.sh', 'conanrun.sh']
hdf5/1.14.5: Calling build()
hdf5/1.14.5: apply_conandata_patches(): No patches defined in conandata
hdf5/1.14.5: Running CMake.configure()
hdf5/1.14.5: RUN: cmake -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE="generators/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="/home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/p" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Release" "/home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/src"
-- Using Conan toolchain: /home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/build/Release/generators/conan_toolchain.cmake
-- Conan toolchain: Setting CMAKE_POSITION_INDEPENDENT_CODE=ON (options.fPIC)
-- Conan toolchain: Defining libcxx as C++ flags: -stdlib=libstdc++
-- Conan toolchain: C++ Standard 17 with extensions ON
-- Conan toolchain: Including CMakeDeps generated conan_cmakedeps_paths.cmake
-- Conan toolchain: Setting BUILD_SHARED_LIBS = OFF
-- The C compiler identification is Clang 19.1.7
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Final: share
-- Looking for include file sys/file.h
-- Looking for include file sys/file.h - found
-- Looking for include files sys/file.h, sys/ioctl.h
-- Looking for include files sys/file.h, sys/ioctl.h - found
-- Looking for 3 include files sys/file.h, ..., sys/resource.h
-- Looking for 3 include files sys/file.h, ..., sys/resource.h - found
-- Looking for 4 include files sys/file.h, ..., sys/socket.h
-- Looking for 4 include files sys/file.h, ..., sys/socket.h - found
-- Looking for 5 include files sys/file.h, ..., sys/stat.h
-- Looking for 5 include files sys/file.h, ..., sys/stat.h - found
-- Looking for 6 include files sys/file.h, ..., sys/time.h
-- Looking for 6 include files sys/file.h, ..., sys/time.h - found
-- Looking for 7 include files sys/file.h, ..., sys/types.h
-- Looking for 7 include files sys/file.h, ..., sys/types.h - found
-- Looking for 8 include files sys/file.h, ..., features.h
-- Looking for 8 include files sys/file.h, ..., features.h - found
-- Looking for 9 include files sys/file.h, ..., dirent.h
-- Looking for 9 include files sys/file.h, ..., dirent.h - found
-- Looking for 10 include files sys/file.h, ..., unistd.h
-- Looking for 10 include files sys/file.h, ..., unistd.h - found
-- Looking for 11 include files sys/file.h, ..., pwd.h
-- Looking for 11 include files sys/file.h, ..., pwd.h - found
-- Looking for 12 include files sys/file.h, ..., pthread.h
-- Looking for 12 include files sys/file.h, ..., pthread.h - found
-- Looking for 13 include files sys/file.h, ..., dlfcn.h
-- Looking for 13 include files sys/file.h, ..., dlfcn.h - found
-- Looking for 14 include files sys/file.h, ..., netinet/in.h
-- Looking for 14 include files sys/file.h, ..., netinet/in.h - found
-- Looking for 15 include files sys/file.h, ..., netdb.h
-- Looking for 15 include files sys/file.h, ..., netdb.h - found
-- Looking for 16 include files sys/file.h, ..., arpa/inet.h
-- Looking for 16 include files sys/file.h, ..., arpa/inet.h - found
-- Looking for include file quadmath.h
-- Looking for include file quadmath.h - not found
-- Looking for ceil in m;
-- Looking for ceil in m; - found
-- Looking for dlopen in dl;m
-- Looking for dlopen in dl;m - found
-- Looking for WSAStartup in ws2_32;m;dl
-- Looking for WSAStartup in ws2_32;m;dl - not found
-- Looking for gethostbyname in wsock32;m;dl
-- Looking for gethostbyname in wsock32;m;dl - not found
-- Looking for gethostname in ucb;m;dl
-- Looking for gethostname in ucb;m;dl - not found
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of char
-- Check size of char - done
-- Check size of short
-- Check size of short - done
-- Check size of int
-- Check size of int - done
-- Check size of unsigned
-- Check size of unsigned - done
-- Check size of long
-- Check size of long - done
-- Check size of long long
-- Check size of long long - done
-- Check size of float
-- Check size of float - done
-- Check size of double
-- Check size of double - done
-- Check size of long double
-- Check size of long double - done
-- Check size of int8_t
-- Check size of int8_t - done
-- Check size of uint8_t
-- Check size of uint8_t - done
-- Check size of int_least8_t
-- Check size of int_least8_t - done
-- Check size of uint_least8_t
-- Check size of uint_least8_t - done
-- Check size of int_fast8_t
-- Check size of int_fast8_t - done
-- Check size of uint_fast8_t
-- Check size of uint_fast8_t - done
-- Check size of int16_t
-- Check size of int16_t - done
-- Check size of uint16_t
-- Check size of uint16_t - done
-- Check size of int_least16_t
-- Check size of int_least16_t - done
-- Check size of uint_least16_t
-- Check size of uint_least16_t - done
-- Check size of int_fast16_t
-- Check size of int_fast16_t - done
-- Check size of uint_fast16_t
-- Check size of uint_fast16_t - done
-- Check size of int32_t
-- Check size of int32_t - done
-- Check size of uint32_t
-- Check size of uint32_t - done
-- Check size of int_least32_t
-- Check size of int_least32_t - done
-- Check size of uint_least32_t
-- Check size of uint_least32_t - done
-- Check size of int_fast32_t
-- Check size of int_fast32_t - done
-- Check size of uint_fast32_t
-- Check size of uint_fast32_t - done
-- Check size of int64_t
-- Check size of int64_t - done
-- Check size of uint64_t
-- Check size of uint64_t - done
-- Check size of int_least64_t
-- Check size of int_least64_t - done
-- Check size of uint_least64_t
-- Check size of uint_least64_t - done
-- Check size of int_fast64_t
-- Check size of int_fast64_t - done
-- Check size of uint_fast64_t
-- Check size of uint_fast64_t - done
-- Check size of size_t
-- Check size of size_t - done
-- Check size of ssize_t
-- Check size of ssize_t - done
-- Check size of ptrdiff_t
-- Check size of ptrdiff_t - done
-- Check size of off_t
-- Check size of off_t - done
-- Check size of time_t
-- Check size of time_t - done
-- Check size of _Bool
-- Check size of _Bool - done
-- Looking for CLOCK_MONOTONIC
-- Looking for CLOCK_MONOTONIC - not found
-- Performing Test H5_HAVE_TM_GMTOFF
-- Performing Test H5_HAVE_TM_GMTOFF - Success
-- Performing Test H5_HAVE___TM_GMTOFF
-- Performing Test H5_HAVE___TM_GMTOFF - Failed
-- Performing Test H5_HAVE_STRUCT_TIMEZONE
-- Performing Test H5_HAVE_STRUCT_TIMEZONE - Failed
-- Looking for gettimeofday
-- Looking for gettimeofday - found
-- Performing Test H5_HAVE_STAT_ST_BLOCKS
-- Performing Test H5_HAVE_STAT_ST_BLOCKS - Success
-- Looking for ioctl
-- Looking for ioctl - found
-- Performing Test H5_HAVE_STRUCT_VIDEOCONFIG
-- Performing Test H5_HAVE_STRUCT_VIDEOCONFIG - Failed
-- Performing Test H5_HAVE_STRUCT_TEXT_INFO
-- Performing Test H5_HAVE_STRUCT_TEXT_INFO - Failed
-- Looking for _getvideoconfig
-- Looking for _getvideoconfig - not found
-- Looking for gettextinfo
-- Looking for gettextinfo - not found
-- Looking for _scrsize
-- Looking for _scrsize - not found
-- Looking for GetConsoleScreenBufferInfo
-- Looking for GetConsoleScreenBufferInfo - not found
-- Looking for TIOCGWINSZ
-- Looking for TIOCGWINSZ - found
-- Looking for TIOCGETD
-- Looking for TIOCGETD - found
-- Looking for alarm
-- Looking for alarm - found
-- Looking for fcntl
-- Looking for fcntl - found
-- Looking for flock
-- Looking for flock - found
-- Looking for fork
-- Looking for fork - found
-- Looking for gethostname
-- Looking for gethostname - found
-- Looking for getrusage
-- Looking for getrusage - found
-- Looking for pread
-- Looking for pread - found
-- Looking for pwrite
-- Looking for pwrite - found
-- Looking for rand_r
-- Looking for rand_r - found
-- Looking for random
-- Looking for random - found
-- Looking for strcasestr
-- Looking for strcasestr - found
-- Looking for strdup
-- Looking for strdup - found
-- Looking for symlink
-- Looking for symlink - found
-- Looking for tmpfile
-- Looking for tmpfile - found
-- Looking for asprintf
-- Looking for asprintf - found
-- Looking for vasprintf
-- Looking for vasprintf - found
-- Looking for waitpid
-- Looking for waitpid - found
-- Looking for sigsetjmp
-- Looking for sigsetjmp - found
-- Looking for clock_gettime
-- Looking for clock_gettime - found
-- Looking for clock_gettime in rt
-- Looking for clock_gettime in rt - found
-- Looking for clock_gettime in posix4
-- Looking for clock_gettime in posix4 - not found
-- Check size of CLOCK_MONOTONIC_COARSE
-- Check size of CLOCK_MONOTONIC_COARSE - done
-- Checking if _Float16 support is available
-- Check size of _Float16
-- Check size of _Float16 - done
-- Looking for FLT16_EPSILON
-- Looking for FLT16_EPSILON - found
-- Looking for FLT16_MIN
-- Looking for FLT16_MIN - found
-- Looking for FLT16_MAX
-- Looking for FLT16_MAX - found
-- Looking for FLT16_MIN_10_EXP
-- Looking for FLT16_MIN_10_EXP - found
-- Looking for FLT16_MAX_10_EXP
-- Looking for FLT16_MAX_10_EXP - found
-- Looking for FLT16_MANT_DIG
-- Looking for FLT16_MANT_DIG - found
-- Looking for fabsf16
-- Looking for fabsf16 - not found
-- Found Perl: /usr/bin/perl (found version "5.26.3")
-- ....All Warnings are enabled
-- Conan: Configuring Targets for zlib/1.3.1
-- Conan: Target declared imported STATIC library 'ZLIB::ZLIB'
-- Filter PLUGIN file is /
-- The CXX compiler identification is Clang 19.1.7
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Warning: Standard CMAKE_CXX_STANDARD value defined in conan_toolchain.cmake to 17 has been modified to 11 by /home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/src/config/cmake/HDFCXXCompilerFlags.cmake
-- ....All Warnings are enabled
-- Configuring done (8.5s)
-- Generating done (0.1s)
-- Build files have been written to: /home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/build/Release

hdf5/1.14.5: Running CMake.build()
hdf5/1.14.5: RUN: cmake --build "/home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/build/Release" -- -j16
[  0%] Building C object src/CMakeFiles/hdf5-static.dir/H5.c.o
[  0%] Building C object src/CMakeFiles/hdf5-static.dir/H5build_settings.c.o
[  0%] Building C object src/CMakeFiles/hdf5-static.dir/H5checksum.c.o
[  1%] Building C object src/CMakeFiles/hdf5-static.dir/H5dbg.c.o
[  1%] Building C object src/CMakeFiles/hdf5-static.dir/H5mpi.c.o
[  1%] Building C object src/CMakeFiles/hdf5-static.dir/H5system.c.o
[  1%] Building C object src/CMakeFiles/hdf5-static.dir/H5timer.c.o
[  2%] Building C object src/CMakeFiles/hdf5-static.dir/H5trace.c.o
[  2%] Building C object src/CMakeFiles/hdf5-static.dir/H5A.c.o
[  2%] Building C object src/CMakeFiles/hdf5-static.dir/H5Abtree2.c.o
[  2%] Building C object src/CMakeFiles/hdf5-static.dir/H5Adense.c.o
[  3%] Building C object src/CMakeFiles/hdf5-static.dir/H5Adeprec.c.o
[  3%] Building C object src/CMakeFiles/hdf5-static.dir/H5Aint.c.o
[  3%] Building C object src/CMakeFiles/hdf5-static.dir/H5Atest.c.o
[  3%] Building C object src/CMakeFiles/hdf5-static.dir/H5AC.c.o
[  4%] Building C object src/CMakeFiles/hdf5-static.dir/H5ACdbg.c.o
[  4%] Building C object src/CMakeFiles/hdf5-static.dir/H5ACmpio.c.o
[  4%] Building C object src/CMakeFiles/hdf5-static.dir/H5ACproxy_entry.c.o
[  4%] Building C object src/CMakeFiles/hdf5-static.dir/H5B.c.o
[  5%] Building C object src/CMakeFiles/hdf5-static.dir/H5Bcache.c.o
[  5%] Building C object src/CMakeFiles/hdf5-static.dir/H5Bdbg.c.o
[  5%] Building C object src/CMakeFiles/hdf5-static.dir/H5B2cache.c.o
[  5%] Building C object src/CMakeFiles/hdf5-static.dir/H5B2.c.o
[  6%] Building C object src/CMakeFiles/hdf5-static.dir/H5B2dbg.c.o
[  6%] Building C object src/CMakeFiles/hdf5-static.dir/H5B2hdr.c.o
[  6%] Building C object src/CMakeFiles/hdf5-static.dir/H5B2int.c.o
[  6%] Building C object src/CMakeFiles/hdf5-static.dir/H5B2internal.c.o
[  7%] Building C object src/CMakeFiles/hdf5-static.dir/H5B2leaf.c.o
[  7%] Building C object src/CMakeFiles/hdf5-static.dir/H5B2stat.c.o
[  7%] Building C object src/CMakeFiles/hdf5-static.dir/H5B2test.c.o
[  7%] Building C object src/CMakeFiles/hdf5-static.dir/H5C.c.o
[  8%] Building C object src/CMakeFiles/hdf5-static.dir/H5Cdbg.c.o
[  8%] Building C object src/CMakeFiles/hdf5-static.dir/H5Centry.c.o
[  8%] Building C object src/CMakeFiles/hdf5-static.dir/H5Cepoch.c.o
[  8%] Building C object src/CMakeFiles/hdf5-static.dir/H5Cimage.c.o
[  9%] Building C object src/CMakeFiles/hdf5-static.dir/H5Cint.c.o
[  9%] Building C object src/CMakeFiles/hdf5-static.dir/H5Clog.c.o
[  9%] Building C object src/CMakeFiles/hdf5-static.dir/H5Clog_json.c.o
[  9%] Building C object src/CMakeFiles/hdf5-static.dir/H5Clog_trace.c.o
[ 10%] Building C object src/CMakeFiles/hdf5-static.dir/H5Cmpio.c.o
[ 10%] Building C object src/CMakeFiles/hdf5-static.dir/H5Cprefetched.c.o
[ 10%] Building C object src/CMakeFiles/hdf5-static.dir/H5Cquery.c.o
[ 11%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ctest.c.o
[ 10%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ctag.c.o
[ 11%] Building C object src/CMakeFiles/hdf5-static.dir/H5CX.c.o
[ 11%] Building C object src/CMakeFiles/hdf5-static.dir/H5D.c.o
[ 11%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dbtree.c.o
[ 12%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dbtree2.c.o
[ 12%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dchunk.c.o
[ 12%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dcompact.c.o
[ 12%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dcontig.c.o
[ 13%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ddbg.c.o
[ 13%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ddeprec.c.o
[ 13%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dearray.c.o
[ 13%] Building C object src/CMakeFiles/hdf5-static.dir/H5Defl.c.o
[ 14%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dfarray.c.o
[ 14%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dfill.c.o
[ 14%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dint.c.o
[ 14%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dio.c.o
[ 14%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dmpio.c.o
[ 15%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dlayout.c.o
[ 15%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dnone.c.o
[ 15%] Building C object src/CMakeFiles/hdf5-static.dir/H5Doh.c.o
[ 16%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dscatgath.c.o
[ 16%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dselect.c.o
[ 16%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dsingle.c.o
[ 16%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dtest.c.o
[ 17%] Building C object src/CMakeFiles/hdf5-static.dir/H5Dvirtual.c.o
[ 17%] Building C object src/CMakeFiles/hdf5-static.dir/H5E.c.o
[ 17%] Building C object src/CMakeFiles/hdf5-static.dir/H5Edeprec.c.o
[ 17%] Building C object src/CMakeFiles/hdf5-static.dir/H5Eint.c.o
[ 18%] Building C object src/CMakeFiles/hdf5-static.dir/H5EA.c.o
[ 18%] Building C object src/CMakeFiles/hdf5-static.dir/H5EAcache.c.o
[ 18%] Building C object src/CMakeFiles/hdf5-static.dir/H5EAdbg.c.o
[ 18%] Building C object src/CMakeFiles/hdf5-static.dir/H5EAdblkpage.c.o
[ 19%] Building C object src/CMakeFiles/hdf5-static.dir/H5EAdblock.c.o
[ 19%] Building C object src/CMakeFiles/hdf5-static.dir/H5EAhdr.c.o
[ 19%] Building C object src/CMakeFiles/hdf5-static.dir/H5EAiblock.c.o
[ 19%] Building C object src/CMakeFiles/hdf5-static.dir/H5EAint.c.o
[ 20%] Building C object src/CMakeFiles/hdf5-static.dir/H5EAsblock.c.o
[ 20%] Building C object src/CMakeFiles/hdf5-static.dir/H5EAstat.c.o
[ 20%] Building C object src/CMakeFiles/hdf5-static.dir/H5EAtest.c.o
[ 20%] Building C object src/CMakeFiles/hdf5-static.dir/H5ES.c.o
[ 21%] Building C object src/CMakeFiles/hdf5-static.dir/H5ESevent.c.o
[ 21%] Building C object src/CMakeFiles/hdf5-static.dir/H5ESint.c.o
[ 21%] Building C object src/CMakeFiles/hdf5-static.dir/H5ESlist.c.o
[ 21%] Building C object src/CMakeFiles/hdf5-static.dir/H5F.c.o
[ 22%] Building C object src/CMakeFiles/hdf5-static.dir/H5Faccum.c.o
[ 22%] Building C object src/CMakeFiles/hdf5-static.dir/H5Fcwfs.c.o
[ 22%] Building C object src/CMakeFiles/hdf5-static.dir/H5Fdbg.c.o
[ 22%] Building C object src/CMakeFiles/hdf5-static.dir/H5Fdeprec.c.o
[ 23%] Building C object src/CMakeFiles/hdf5-static.dir/H5Fefc.c.o
[ 23%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ffake.c.o
[ 23%] Building C object src/CMakeFiles/hdf5-static.dir/H5Fint.c.o
[ 23%] Building C object src/CMakeFiles/hdf5-static.dir/H5Fio.c.o
[ 24%] Building C object src/CMakeFiles/hdf5-static.dir/H5Fmount.c.o
[ 24%] Building C object src/CMakeFiles/hdf5-static.dir/H5Fmpi.c.o
[ 24%] Building C object src/CMakeFiles/hdf5-static.dir/H5Fquery.c.o
[ 24%] Building C object src/CMakeFiles/hdf5-static.dir/H5Fsfile.c.o
[ 25%] Building C object src/CMakeFiles/hdf5-static.dir/H5Fspace.c.o
[ 25%] Building C object src/CMakeFiles/hdf5-static.dir/H5Fsuper.c.o
[ 25%] Building C object src/CMakeFiles/hdf5-static.dir/H5Fsuper_cache.c.o
[ 25%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ftest.c.o
[ 26%] Building C object src/CMakeFiles/hdf5-static.dir/H5FA.c.o
[ 26%] Building C object src/CMakeFiles/hdf5-static.dir/H5FAcache.c.o
[ 26%] Building C object src/CMakeFiles/hdf5-static.dir/H5FAdbg.c.o
[ 26%] Building C object src/CMakeFiles/hdf5-static.dir/H5FAdblkpage.c.o
[ 27%] Building C object src/CMakeFiles/hdf5-static.dir/H5FAdblock.c.o
[ 27%] Building C object src/CMakeFiles/hdf5-static.dir/H5FAhdr.c.o
[ 27%] Building C object src/CMakeFiles/hdf5-static.dir/H5FAint.c.o
[ 27%] Building C object src/CMakeFiles/hdf5-static.dir/H5FAstat.c.o
[ 28%] Building C object src/CMakeFiles/hdf5-static.dir/H5FAtest.c.o
[ 28%] Building C object src/CMakeFiles/hdf5-static.dir/H5FD.c.o
[ 28%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDcore.c.o
[ 28%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDdirect.c.o
[ 29%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDfamily.c.o
[ 29%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDhdfs.c.o
[ 29%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDlog.c.o
[ 29%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDint.c.o
[ 30%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDmirror.c.o
[ 30%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDmpi.c.o
[ 30%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDmpio.c.o
[ 30%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDmulti.c.o
[ 31%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDonion.c.o
[ 31%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDonion_header.c.o
[ 31%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDonion_history.c.o
[ 31%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDonion_index.c.o
[ 32%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDperform.c.o
[ 32%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDros3.c.o
[ 32%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDs3comms.c.o
[ 32%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDsec2.c.o
[ 33%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDspace.c.o
[ 33%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDsplitter.c.o
[ 33%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDstdio.c.o
[ 33%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDtest.c.o
[ 34%] Building C object src/CMakeFiles/hdf5-static.dir/H5FDwindows.c.o
[ 34%] Building C object src/CMakeFiles/hdf5-static.dir/H5FL.c.o
[ 34%] Building C object src/CMakeFiles/hdf5-static.dir/H5FO.c.o
[ 34%] Building C object src/CMakeFiles/hdf5-static.dir/H5FS.c.o
[ 35%] Building C object src/CMakeFiles/hdf5-static.dir/H5FScache.c.o
[ 35%] Building C object src/CMakeFiles/hdf5-static.dir/H5FSdbg.c.o
[ 35%] Building C object src/CMakeFiles/hdf5-static.dir/H5FSsection.c.o
[ 35%] Building C object src/CMakeFiles/hdf5-static.dir/H5FSint.c.o
[ 36%] Building C object src/CMakeFiles/hdf5-static.dir/H5FSstat.c.o
[ 36%] Building C object src/CMakeFiles/hdf5-static.dir/H5FStest.c.o
[ 36%] Building C object src/CMakeFiles/hdf5-static.dir/H5G.c.o
[ 36%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gbtree2.c.o
[ 37%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gcache.c.o
[ 37%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gcompact.c.o
[ 37%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gdense.c.o
[ 37%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gdeprec.c.o
[ 38%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gent.c.o
[ 38%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gint.c.o
[ 38%] Building C object src/CMakeFiles/hdf5-static.dir/H5Glink.c.o
[ 38%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gloc.c.o
[ 39%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gname.c.o
[ 39%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gnode.c.o
[ 39%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gobj.c.o
[ 39%] Building C object src/CMakeFiles/hdf5-static.dir/H5Goh.c.o
[ 40%] Building C object src/CMakeFiles/hdf5-static.dir/H5Groot.c.o
[ 40%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gstab.c.o
[ 40%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gtest.c.o
[ 40%] Building C object src/CMakeFiles/hdf5-static.dir/H5Gtraverse.c.o
[ 41%] Building C object src/CMakeFiles/hdf5-static.dir/H5HF.c.o
[ 41%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFbtree2.c.o
[ 41%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFcache.c.o
[ 41%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFdbg.c.o
[ 42%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFdblock.c.o
[ 42%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFdtable.c.o
[ 42%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFhdr.c.o
[ 42%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFhuge.c.o
[ 42%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFiter.c.o
[ 42%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFman.c.o
[ 43%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFiblock.c.o
[ 43%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFsection.c.o
[ 44%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFspace.c.o
[ 44%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFstat.c.o
[ 44%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFtest.c.o
[ 45%] Building C object src/CMakeFiles/hdf5-static.dir/H5HG.c.o
[ 45%] Building C object src/CMakeFiles/hdf5-static.dir/H5HFtiny.c.o
[ 45%] Building C object src/CMakeFiles/hdf5-static.dir/H5HGcache.c.o
[ 45%] Building C object src/CMakeFiles/hdf5-static.dir/H5HGdbg.c.o
[ 45%] Building C object src/CMakeFiles/hdf5-static.dir/H5HGquery.c.o
[ 46%] Building C object src/CMakeFiles/hdf5-static.dir/H5HL.c.o
[ 46%] Building C object src/CMakeFiles/hdf5-static.dir/H5HLcache.c.o
[ 46%] Building C object src/CMakeFiles/hdf5-static.dir/H5HLdbg.c.o
[ 46%] Building C object src/CMakeFiles/hdf5-static.dir/H5HLdblk.c.o
[ 47%] Building C object src/CMakeFiles/hdf5-static.dir/H5HLint.c.o
[ 47%] Building C object src/CMakeFiles/hdf5-static.dir/H5HLprfx.c.o
[ 47%] Building C object src/CMakeFiles/hdf5-static.dir/H5Idbg.c.o
[ 47%] Building C object src/CMakeFiles/hdf5-static.dir/H5I.c.o
[ 48%] Building C object src/CMakeFiles/hdf5-static.dir/H5Iint.c.o
[ 48%] Building C object src/CMakeFiles/hdf5-static.dir/H5Itest.c.o
[ 48%] Building C object src/CMakeFiles/hdf5-static.dir/H5L.c.o
[ 48%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ldeprec.c.o
[ 49%] Building C object src/CMakeFiles/hdf5-static.dir/H5Lexternal.c.o
[ 49%] Building C object src/CMakeFiles/hdf5-static.dir/H5Lint.c.o
[ 49%] Building C object src/CMakeFiles/hdf5-static.dir/H5M.c.o
[ 49%] Building C object src/CMakeFiles/hdf5-static.dir/H5MF.c.o
[ 49%] Building C object src/CMakeFiles/hdf5-static.dir/H5MFdbg.c.o
[ 50%] Building C object src/CMakeFiles/hdf5-static.dir/H5MFaggr.c.o
[ 50%] Building C object src/CMakeFiles/hdf5-static.dir/H5MFsection.c.o
[ 50%] Building C object src/CMakeFiles/hdf5-static.dir/H5MM.c.o
[ 51%] Building C object src/CMakeFiles/hdf5-static.dir/H5O.c.o
[ 51%] Building C object src/CMakeFiles/hdf5-static.dir/H5Oainfo.c.o
[ 51%] Building C object src/CMakeFiles/hdf5-static.dir/H5Oalloc.c.o
[ 51%] Building C object src/CMakeFiles/hdf5-static.dir/H5Oattr.c.o
/home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/src/src/H5MF.c:2078:84: warning: arithmetic between different enumeration types ('H5F_mem_page_t' (aka 'enum H5F_mem_page_t') and 'enum H5F_mem_t') [-Wenum-enum-conversion]
 2078 |                                                                          : ((ptype % H5FD_MEM_NTYPES) + 1));
      |                                                                              ~~~~~ ^ ~~~~~~~~~~~~~~~
[ 52%] Building C object src/CMakeFiles/hdf5-static.dir/H5Oattribute.c.o
/home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/src/src/H5MF.c:2318:50: warning: arithmetic between different enumeration types ('H5F_mem_page_t' (aka 'enum H5F_mem_page_t') and 'enum H5F_mem_t') [-Wenum-enum-conversion]
 2318 |             end_type = (H5F_mem_page_t)(end_type + H5FD_MEM_NTYPES);
      |                                         ~~~~~~~~ ^ ~~~~~~~~~~~~~~~
/home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/src/src/H5MF.c:2372:38: warning: arithmetic between different enumeration types ('H5F_mem_page_t' (aka 'enum H5F_mem_page_t') and 'enum H5F_mem_t') [-Wenum-enum-conversion]
 2372 |             ty = (H5F_mem_page_t)(ty + H5FD_MEM_NTYPES - 2);
      |                                   ~~ ^ ~~~~~~~~~~~~~~~
[ 52%] Building C object src/CMakeFiles/hdf5-static.dir/H5Obogus.c.o
[ 52%] Building C object src/CMakeFiles/hdf5-static.dir/H5Obtreek.c.o
[ 52%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ocache.c.o
[ 53%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ocache_image.c.o
[ 53%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ochunk.c.o
[ 53%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ocont.c.o
[ 53%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ocopy.c.o
[ 54%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ocopy_ref.c.o
[ 54%] Building C object src/CMakeFiles/hdf5-static.dir/H5Odbg.c.o
[ 54%] Building C object src/CMakeFiles/hdf5-static.dir/H5Odeprec.c.o
[ 54%] Building C object src/CMakeFiles/hdf5-static.dir/H5Odrvinfo.c.o
[ 55%] Building C object src/CMakeFiles/hdf5-static.dir/H5Odtype.c.o
[ 55%] Building C object src/CMakeFiles/hdf5-static.dir/H5Oefl.c.o
[ 55%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ofill.c.o
[ 55%] Building C object src/CMakeFiles/hdf5-static.dir/H5Oflush.c.o
[ 56%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ofsinfo.c.o
[ 56%] Building C object src/CMakeFiles/hdf5-static.dir/H5Oginfo.c.o
[ 56%] Building C object src/CMakeFiles/hdf5-static.dir/H5Olayout.c.o
[ 56%] Building C object src/CMakeFiles/hdf5-static.dir/H5Oint.c.o
[ 57%] Building C object src/CMakeFiles/hdf5-static.dir/H5Olinfo.c.o
[ 57%] Building C object src/CMakeFiles/hdf5-static.dir/H5Olink.c.o
3 warnings generated.
[ 57%] Building C object src/CMakeFiles/hdf5-static.dir/H5Omessage.c.o
[ 57%] Building C object src/CMakeFiles/hdf5-static.dir/H5Omtime.c.o
[ 58%] Building C object src/CMakeFiles/hdf5-static.dir/H5Oname.c.o
[ 58%] Building C object src/CMakeFiles/hdf5-static.dir/H5Onull.c.o
[ 58%] Building C object src/CMakeFiles/hdf5-static.dir/H5Opline.c.o
[ 58%] Building C object src/CMakeFiles/hdf5-static.dir/H5Orefcount.c.o
[ 59%] Building C object src/CMakeFiles/hdf5-static.dir/H5Osdspace.c.o
[ 59%] Building C object src/CMakeFiles/hdf5-static.dir/H5Oshared.c.o
[ 59%] Building C object src/CMakeFiles/hdf5-static.dir/H5Oshmesg.c.o
[ 59%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ostab.c.o
[ 60%] Building C object src/CMakeFiles/hdf5-static.dir/H5Otest.c.o
[ 60%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ounknown.c.o
[ 60%] Building C object src/CMakeFiles/hdf5-static.dir/H5P.c.o
[ 60%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pacpl.c.o
[ 61%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pdapl.c.o
[ 61%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pdcpl.c.o
[ 61%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pdeprec.c.o
[ 61%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pdxpl.c.o
[ 62%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pencdec.c.o
[ 62%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pfapl.c.o
[ 62%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pfcpl.c.o
[ 62%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pfmpl.c.o
[ 63%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pgcpl.c.o
[ 63%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pint.c.o
[ 63%] Building C object src/CMakeFiles/hdf5-static.dir/H5Plapl.c.o
[ 63%] Building C object src/CMakeFiles/hdf5-static.dir/H5Plcpl.c.o
[ 64%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pmapl.c.o
[ 64%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pmcpl.c.o
[ 64%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pocpl.c.o
[ 64%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pocpypl.c.o
[ 65%] Building C object src/CMakeFiles/hdf5-static.dir/H5Pstrcpl.c.o
[ 65%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ptest.c.o
[ 65%] Building C object src/CMakeFiles/hdf5-static.dir/H5PB.c.o
[ 65%] Building C object src/CMakeFiles/hdf5-static.dir/H5PL.c.o
[ 66%] Building C object src/CMakeFiles/hdf5-static.dir/H5PLint.c.o
[ 66%] Building C object src/CMakeFiles/hdf5-static.dir/H5PLpath.c.o
[ 66%] Building C object src/CMakeFiles/hdf5-static.dir/H5PLplugin_cache.c.o
[ 66%] Building C object src/CMakeFiles/hdf5-static.dir/H5R.c.o
[ 67%] Building C object src/CMakeFiles/hdf5-static.dir/H5Rdeprec.c.o
[ 67%] Building C object src/CMakeFiles/hdf5-static.dir/H5Rint.c.o
[ 67%] Building C object src/CMakeFiles/hdf5-static.dir/H5UC.c.o
[ 67%] Building C object src/CMakeFiles/hdf5-static.dir/H5RS.c.o
[ 68%] Building C object src/CMakeFiles/hdf5-static.dir/H5S.c.o
[ 68%] Building C object src/CMakeFiles/hdf5-static.dir/H5Sall.c.o
[ 68%] Building C object src/CMakeFiles/hdf5-static.dir/H5Sdbg.c.o
[ 68%] Building C object src/CMakeFiles/hdf5-static.dir/H5Sdeprec.c.o
[ 69%] Building C object src/CMakeFiles/hdf5-static.dir/H5Shyper.c.o
[ 69%] Building C object src/CMakeFiles/hdf5-static.dir/H5Smpio.c.o
[ 69%] Building C object src/CMakeFiles/hdf5-static.dir/H5Snone.c.o
[ 69%] Building C object src/CMakeFiles/hdf5-static.dir/H5Spoint.c.o
[ 70%] Building C object src/CMakeFiles/hdf5-static.dir/H5Sselect.c.o
[ 70%] Building C object src/CMakeFiles/hdf5-static.dir/H5Stest.c.o
[ 70%] Building C object src/CMakeFiles/hdf5-static.dir/H5SL.c.o
[ 70%] Building C object src/CMakeFiles/hdf5-static.dir/H5SM.c.o
[ 71%] Building C object src/CMakeFiles/hdf5-static.dir/H5SMbtree2.c.o
[ 71%] Building C object src/CMakeFiles/hdf5-static.dir/H5SMcache.c.o
[ 71%] Building C object src/CMakeFiles/hdf5-static.dir/H5SMmessage.c.o
[ 71%] Building C object src/CMakeFiles/hdf5-static.dir/H5SMtest.c.o
[ 72%] Building C object src/CMakeFiles/hdf5-static.dir/H5T.c.o
[ 72%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tarray.c.o
[ 72%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tbit.c.o
[ 72%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tcommit.c.o
[ 72%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tconv.c.o
[ 72%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tconv_integer.c.o
[ 73%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tcompound.c.o
[ 73%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tconv_float.c.o
[ 74%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tconv_string.c.o
[ 74%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tconv_bitfield.c.o
[ 74%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tconv_compound.c.o
[ 74%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tconv_reference.c.o
[ 75%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tconv_enum.c.o
[ 75%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tconv_vlen.c.o
[ 75%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tconv_array.c.o
[ 75%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tcset.c.o
[ 76%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tdbg.c.o
[ 76%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tdeprec.c.o
[ 76%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tenum.c.o
[ 76%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tfields.c.o
[ 77%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tfixed.c.o
[ 77%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tfloat.c.o
[ 77%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tinit_float.c.o
[ 77%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tnative.c.o
[ 78%] Building C object src/CMakeFiles/hdf5-static.dir/H5Toffset.c.o
[ 78%] Building C object src/CMakeFiles/hdf5-static.dir/H5Toh.c.o
[ 78%] Building C object src/CMakeFiles/hdf5-static.dir/H5Topaque.c.o
[ 78%] Building C object src/CMakeFiles/hdf5-static.dir/H5Torder.c.o
[ 79%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tpad.c.o
[ 79%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tprecis.c.o
[ 79%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tref.c.o
[ 79%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tstrpad.c.o
[ 80%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tvisit.c.o
[ 80%] Building C object src/CMakeFiles/hdf5-static.dir/H5Tvlen.c.o
[ 80%] Building C object src/CMakeFiles/hdf5-static.dir/H5VL.c.o
[ 80%] Building C object src/CMakeFiles/hdf5-static.dir/H5TS.c.o
[ 81%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLcallback.c.o
[ 81%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLdyn_ops.c.o
[ 81%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLint.c.o
[ 81%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLnative.c.o
[ 82%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLnative_attr.c.o
[ 82%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLnative_blob.c.o
[ 82%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLnative_dataset.c.o
[ 82%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLnative_datatype.c.o
[ 83%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLnative_file.c.o
[ 83%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLnative_group.c.o
[ 83%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLnative_link.c.o
[ 83%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLnative_introspect.c.o
[ 83%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLnative_token.c.o
[ 84%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLnative_object.c.o
[ 84%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLpassthru.c.o
[ 84%] Building C object src/CMakeFiles/hdf5-static.dir/H5VLtest.c.o
[ 85%] Building C object src/CMakeFiles/hdf5-static.dir/H5VM.c.o
[ 85%] Building C object src/CMakeFiles/hdf5-static.dir/H5WB.c.o
[ 85%] Building C object src/CMakeFiles/hdf5-static.dir/H5Zdeflate.c.o
[ 85%] Building C object src/CMakeFiles/hdf5-static.dir/H5Z.c.o
[ 86%] Building C object src/CMakeFiles/hdf5-static.dir/H5Zfletcher32.c.o
[ 86%] Building C object src/CMakeFiles/hdf5-static.dir/H5Znbit.c.o
[ 86%] Building C object src/CMakeFiles/hdf5-static.dir/H5Zscaleoffset.c.o
[ 86%] Building C object src/CMakeFiles/hdf5-static.dir/H5Zshuffle.c.o
[ 87%] Building C object src/CMakeFiles/hdf5-static.dir/H5Zszip.c.o
[ 87%] Building C object src/CMakeFiles/hdf5-static.dir/H5Ztrans.c.o
[ 87%] Linking C static library libhdf5.a
[ 87%] Built target hdf5-static
[ 87%] Building C object utils/mirror_vfd/CMakeFiles/mirror_server_stop.dir/mirror_server_stop.c.o
[ 87%] Building C object utils/mirror_vfd/CMakeFiles/mirror_server.dir/mirror_remote.c.o
[ 88%] Building C object hl/src/CMakeFiles/hdf5_hl-static.dir/H5DO.c.o
[ 88%] Building C object utils/mirror_vfd/CMakeFiles/mirror_server.dir/mirror_server.c.o
[ 88%] Building C object utils/mirror_vfd/CMakeFiles/mirror_server.dir/mirror_writer.c.o
[ 89%] Building C object hl/src/CMakeFiles/hdf5_hl-static.dir/H5DS.c.o
[ 89%] Building C object hl/src/CMakeFiles/hdf5_hl-static.dir/H5IM.c.o
[ 89%] Building CXX object c++/src/CMakeFiles/hdf5_cpp-static.dir/H5AbstractDs.cpp.o
[ 90%] Building CXX object c++/src/CMakeFiles/hdf5_cpp-static.dir/H5ArrayType.cpp.o
[ 90%] Building C object hl/src/CMakeFiles/hdf5_hl-static.dir/H5LT.c.o
[ 90%] Building CXX object c++/src/CMakeFiles/hdf5_cpp-static.dir/H5AtomType.cpp.o
[ 90%] Building CXX object c++/src/CMakeFiles/hdf5_cpp-static.dir/H5Attribute.cpp.o
[ 91%] Building C object hl/src/CMakeFiles/hdf5_hl-static.dir/H5PT.c.o
[ 90%] Building C object hl/src/CMakeFiles/hdf5_hl-static.dir/H5LTanalyze.c.o
[ 91%] Building C object hl/src/CMakeFiles/hdf5_hl-static.dir/H5LTparse.c.o
[ 91%] Building C object hl/src/CMakeFiles/hdf5_hl-static.dir/H5TB.c.o
/home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/src/c++/src/H5AbstractDs.cpp:13:10: fatal error: 'string' file not found
   13 | #include <string>
      |          ^~~~~~~~
/home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/src/c++/src/H5AtomType.cpp:13:10: fatal error: 'string' file not found
   13 | #include <string>
      |          ^~~~~~~~
/home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/src/c++/src/H5ArrayType.cpp:13:10: fatal error: 'string' file not found
   13 | #include <string>
      |          ^~~~~~~~
/home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/src/c++/src/H5Attribute.cpp:13:10: fatal error: 'cstdlib' file not found
   13 | #include <cstdlib>
      |          ^~~~~~~~~
1 error generated.
1 error generated.
gmake[2]: *** [c++/src/CMakeFiles/hdf5_cpp-static.dir/build.make:93: c++/src/CMakeFiles/hdf5_cpp-static.dir/H5ArrayType.cpp.o] Error 1
gmake[2]: *** Waiting for unfinished jobs....
gmake[2]: *** [c++/src/CMakeFiles/hdf5_cpp-static.dir/build.make:107: c++/src/CMakeFiles/hdf5_cpp-static.dir/H5AtomType.cpp.o] Error 1
[ 91%] Building C object hl/src/CMakeFiles/hdf5_hl-static.dir/H5LD.c.o
1 error generated.
gmake[2]: *** [c++/src/CMakeFiles/hdf5_cpp-static.dir/build.make:79: c++/src/CMakeFiles/hdf5_cpp-static.dir/H5AbstractDs.cpp.o] Error 1
[ 92%] Linking C executable mirror_server_stop
1 error generated.
gmake[2]: *** [c++/src/CMakeFiles/hdf5_cpp-static.dir/build.make:121: c++/src/CMakeFiles/hdf5_cpp-static.dir/H5Attribute.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:450: c++/src/CMakeFiles/hdf5_cpp-static.dir/all] Error 2
gmake[1]: *** Waiting for unfinished jobs....
[ 92%] Linking C executable mirror_server
[ 92%] Built target mirror_server_stop
[ 92%] Built target mirror_server
[ 93%] Linking C static library libhdf5_hl.a
[ 93%] Built target hdf5_hl-static
gmake: *** [Makefile:136: all] Error 2

hdf5/1.14.5: ERROR: 
Package 'cadfbce37b6206055ba85ecacc65e7f1ab3b640b' build failed
hdf5/1.14.5: WARN: Build folder /home/mbarbier/.conan2/p/b/hdf5cbd1bd2ed4694/b/build/Release
ERROR: hdf5/1.14.5: Error in build() method, line 186
        cmake.build()
        ConanException: Error 2 while executing
```

</details>

<details><summary>Make VERBOSE=1</summary>

```
/home/mbarbier/.conan2/p/b/cmake4953afb13739c/p/bin/cmake -S/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src -B/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release --check-build-system CMakeFiles/Makefile.cmake 0
/home/mbarbier/.conan2/p/b/cmake4953afb13739c/p/bin/cmake -E cmake_progress_start /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release/CMakeFiles /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release//CMakeFiles/progress.marks
make  -f CMakeFiles/Makefile2 all
make[1]: Entering directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
make  -f src/CMakeFiles/hdf5-static.dir/build.make src/CMakeFiles/hdf5-static.dir/depend
make[2]: Entering directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
cd /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release && /home/mbarbier/.conan2/p/b/cmake4953afb13739c/p/bin/cmake -E cmake_depends "Unix Makefiles" /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src/src /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release/src /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release/src/CMakeFiles/hdf5-static.dir/DependInfo.cmake "--color="
make[2]: Leaving directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
make  -f src/CMakeFiles/hdf5-static.dir/build.make src/CMakeFiles/hdf5-static.dir/build
make[2]: Entering directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
make[2]: Nothing to be done for 'src/CMakeFiles/hdf5-static.dir/build'.
make[2]: Leaving directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
[ 87%] Built target hdf5-static
make  -f utils/mirror_vfd/CMakeFiles/mirror_server.dir/build.make utils/mirror_vfd/CMakeFiles/mirror_server.dir/depend
make[2]: Entering directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
cd /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release && /home/mbarbier/.conan2/p/b/cmake4953afb13739c/p/bin/cmake -E cmake_depends "Unix Makefiles" /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src/utils/mirror_vfd /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release/utils/mirror_vfd /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release/utils/mirror_vfd/CMakeFiles/mirror_server.dir/DependInfo.cmake "--color="
make[2]: Leaving directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
make  -f utils/mirror_vfd/CMakeFiles/mirror_server.dir/build.make utils/mirror_vfd/CMakeFiles/mirror_server.dir/build
make[2]: Entering directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
make[2]: Nothing to be done for 'utils/mirror_vfd/CMakeFiles/mirror_server.dir/build'.
make[2]: Leaving directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
[ 88%] Built target mirror_server
make  -f utils/mirror_vfd/CMakeFiles/mirror_server_stop.dir/build.make utils/mirror_vfd/CMakeFiles/mirror_server_stop.dir/depend
make[2]: Entering directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
cd /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release && /home/mbarbier/.conan2/p/b/cmake4953afb13739c/p/bin/cmake -E cmake_depends "Unix Makefiles" /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src/utils/mirror_vfd /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release/utils/mirror_vfd /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release/utils/mirror_vfd/CMakeFiles/mirror_server_stop.dir/DependInfo.cmake "--color="
make[2]: Leaving directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
make  -f utils/mirror_vfd/CMakeFiles/mirror_server_stop.dir/build.make utils/mirror_vfd/CMakeFiles/mirror_server_stop.dir/build
make[2]: Entering directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
make[2]: Nothing to be done for 'utils/mirror_vfd/CMakeFiles/mirror_server_stop.dir/build'.
make[2]: Leaving directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
[ 89%] Built target mirror_server_stop
make  -f hl/src/CMakeFiles/hdf5_hl-static.dir/build.make hl/src/CMakeFiles/hdf5_hl-static.dir/depend
make[2]: Entering directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
cd /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release && /home/mbarbier/.conan2/p/b/cmake4953afb13739c/p/bin/cmake -E cmake_depends "Unix Makefiles" /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src/hl/src /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release/hl/src /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release/hl/src/CMakeFiles/hdf5_hl-static.dir/DependInfo.cmake "--color="
make[2]: Leaving directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
make  -f hl/src/CMakeFiles/hdf5_hl-static.dir/build.make hl/src/CMakeFiles/hdf5_hl-static.dir/build
make[2]: Entering directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
make[2]: Nothing to be done for 'hl/src/CMakeFiles/hdf5_hl-static.dir/build'.
make[2]: Leaving directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
[ 92%] Built target hdf5_hl-static
make  -f c++/src/CMakeFiles/hdf5_cpp-static.dir/build.make c++/src/CMakeFiles/hdf5_cpp-static.dir/depend
make[2]: Entering directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
cd /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release && /home/mbarbier/.conan2/p/b/cmake4953afb13739c/p/bin/cmake -E cmake_depends "Unix Makefiles" /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src/c++/src /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release/c++/src /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release/c++/src/CMakeFiles/hdf5_cpp-static.dir/DependInfo.cmake "--color="
make[2]: Leaving directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
make  -f c++/src/CMakeFiles/hdf5_cpp-static.dir/build.make c++/src/CMakeFiles/hdf5_cpp-static.dir/build
make[2]: Entering directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
[ 92%] Building CXX object c++/src/CMakeFiles/hdf5_cpp-static.dir/H5AbstractDs.cpp.o
cd /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release/c++/src && /usr/bin/clang++ -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE -D_LARGEFILE_SOURCE -D_POSIX_C_SOURCE=200809L -I/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src/src -I/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src/src/H5FDsubfiling -I/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release/src -stdlib=libstdc++ -O3 -DNDEBUG -std=c++11 -fPIC -Wall -Warray-bounds -Wcast-qual -Wconversion -Wdouble-promotion -Wextra -Wformat=2 -Wframe-larger-than=16384 -Wimplicit-fallthrough -Wnull-dereference -Wunused-const-variable -Wwrite-strings -Wpedantic -Wvolatile-register-var -Wno-c++-compat -Wno-missing-noreturn -MD -MT c++/src/CMakeFiles/hdf5_cpp-static.dir/H5AbstractDs.cpp.o -MF CMakeFiles/hdf5_cpp-static.dir/H5AbstractDs.cpp.o.d -o CMakeFiles/hdf5_cpp-static.dir/H5AbstractDs.cpp.o -c /home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src/c++/src/H5AbstractDs.cpp
/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/src/c++/src/H5AbstractDs.cpp:13:10: fatal error: 'string' file not found
   13 | #include <string>
      |          ^~~~~~~~
1 error generated.
make[2]: *** [c++/src/CMakeFiles/hdf5_cpp-static.dir/build.make:79: c++/src/CMakeFiles/hdf5_cpp-static.dir/H5AbstractDs.cpp.o] Error 1
make[2]: Leaving directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
make[1]: *** [CMakeFiles/Makefile2:450: c++/src/CMakeFiles/hdf5_cpp-static.dir/all] Error 2
make[1]: Leaving directory '/home/mbarbier/.conan2/p/b/hdf5aaff69a3e8fc3/b/build/Release'
make: *** [Makefile:136: all] Error 2
```

</details>

--- a/recipes/hdf5/all/conandata.yml
+++ b/recipes/hdf5/all/conandata.yml
@@ -1,7 +1,4 @@
 sources:
-  "1.14.0":
-    url: "https://github.com/HDFGroup/hdf5/archive/hdf5-1_14_0.tar.gz"
-    sha256: "0f2ec13c1bf6e9f506fd03681c83a476d722e7479cad9ce6f8585a26c317d0ad"
   "1.14.6":
     url: "https://github.com/HDFGroup/hdf5/archive/refs/tags/hdf5_1.14.6.tar.gz"
     sha256: "09ee1c671a87401a5201c06106650f62badeea5a3b3941e9b1e2e1e08317357f"

